htdocs/core/class/commonobject.class.php

```diff
@@ -10118,15 +10124,15 @@ public function deleteByParentField($parentId = 0, $parentField = '', $filter =
 				$error++;
 			} else {
 				while ($obj = $this->db->fetch_object($resql)) {
-					$result = $this->fetch($obj->rowid);
+					$result = $this->fetchCommon($obj->rowid);
```

Comment:
The method fetch does not exists because it is defined into parent. When a parent instance call deleteByParentField, we must delete the fetch of the parent instance.